src/include/sof/probe/probe.h

```diff
@@ -74,12 +66,12 @@ int probe_dma_info(struct sof_ipc_probe_info_params *data, uint32_t max_size);
 int probe_dma_remove(uint32_t count, uint32_t *stream_tag);
 
 /*
- * \brief Set probe points
+ * \brief Get info about connected injection DMAs
  *
- * param[in] count - number of probe points configured this call
- * param[in] probe - array of size 'count' with configuration of probe points
+ * param[in,out] data - reply to write data to
+ * param[in] max_size - maximum number of bytes available in data
  */
-int probe_point_add(uint32_t count, struct probe_point *probe);
+int probe_dma_info(struct sof_ipc_probe_info_params *data, uint32_t max_size);
```

Comment:
why did you reorder these? They even were in alphabetic order.

Reply:
Not needed, reverted